adapters/repos/db/docid/scan.go

```diff
@@ -85,13 +88,15 @@ func (os *objectScannerLSM) scan() error {
 		if res == nil {
 			continue
 		}
-
-		elem, err := storobj.FromBinary(res)
-		if err != nil {
-			return errors.Wrapf(err, "unmarshal data object")
+		var properties models.PropertySchema = nil
```

Comment:
I think it's sufficient just to do `var properties models.PropertySchema` without explicit `= nil` assignment, bc by default `PropertySchema` will be nil